types/options.d.ts

```diff
@@ -65,5 +65,6 @@ interface PointAnnotationOptions extends CoreAnnotationOptions {
 export interface AnnotationPluginOptions extends AnnotationEvents {
 	annotations: AnnotationOptions[] | Record<string, AnnotationOptions>,
 	dblClickSpeed?: Scriptable<number, PartialEventContext>,
-	drawTime?: Scriptable<DrawTime, PartialEventContext>,
+  drawTime?: Scriptable<DrawTime, PartialEventContext>,
+  animations: Record<string, unknown>,
```

Comment:
Since this commit, I have the following error in my project:
> error TS2741: Property 'animations' is missing in type '{ annotations: { [key: string]: AnnotationOptions<keyof AnnotationTypeRegistry>; }; }' but required in type 'AnnotationPluginOptions'.

Shouldn't the `animations` property be optional ?
```diff
- animations: Record<string, unknown>,
+ animations?: Record<string, unknown>,
```

I don't find any documentation on this property in https://www.chartjs.org/chartjs-plugin-annotation/latest/ ; am I missing something ?

Thanks!